msal/build.gradle

```diff
@@ -280,7 +281,7 @@ dependencies {
     testImplementation "org.mockito:mockito-inline:$rootProject.ext.mockitoCoreVersion"
     testImplementation ("org.robolectric:robolectric:$rootProject.ext.robolectricVersion")
     // For mockito!
-    testImplementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.13'
+    testImplementation "cz.msebera.android:httpclient:$rootProject.ext.msebraApacheHttpClientVersion"
```

Comment:
implementation should carry over to test dependencies as well...why do we need this explicitly?